src/rust/ide/view/graph-editor/src/profiling_icon.rs

```diff
@@ -0,0 +1,56 @@
+use crate::prelude::*;
```

Comment:
1. No docs
2. No ections
3. Too long lines
4. No newline at the end of the file
5. misaligned lines